### PR TITLE
 Updated Shell scripts 

### PR DESCRIPTION
**Pull request by mmphego**
- Double quote to prevent globbing and word splitting.
- Changed bash shebang from `#!/bin/bash` to `#!/usr/bin/env bash` for [portability](https://en.wikipedia.org/wiki/Shebang_%28Unix%29#Portability)

Resolves #

### CODE REVIEW

File: script/build/linux-entrypoint [hunk 1]
```diff
@@ -5,11 +5,15 @@ set -ex
...
+chmod 777 "$(pwd)"/dist
+
+if [ ! -f "${VENV}/bin/pip" ]; then
+    "${VENV}"/bin/pip install -q -r requirements-build.txt
+    ./script/build/write-git-sha
+    su -c "${VENV}/bin/pyinstaller docker-compose.spec" user
+    mv dist/docker-compose "${TARGET}"
+    "${TARGET}" version
+else
+    echo "Python-Pip is not installed in your Env"
```
Comment (hartwork) [inline script/build/linux-entrypoint]: `>&2` and `exit 1` might be missing here. What do you think?

File: script/build/linux-entrypoint [hunk 2]
```diff
@@ -5,11 +5,15 @@ set -ex
...
+fi
```
Comment (hartwork) [inline script/build/linux-entrypoint]: Looks like a trailing newline is missing here.

File: script/build/linux-entrypoint (hunk 1 above)
Comment (mmphego) [reply script/build/linux-entrypoint]: Thanks for the heads up - I have resolved this issue!

File: script/build/linux-entrypoint (hunk 2 above)
Comment (mmphego) [reply script/build/linux-entrypoint]: Added trailing newline.

File: script/release/setup-venv.sh [hunk 3]
```diff
@@ -1,40 +1,39 @@
...
-$PYTHONBIN -m venv $VENV_DIR $VENV_FLAGS
+$PYTHONBIN -m venv "${VENV_DIR}" "${VENV_FLAGS}"
 
-VENV_PYTHONBIN=$VENV_DIR/bin/python
+VENV_PYTHONBIN="${VENV_DIR}"/bin/python
 
 if debian_based; then
-  curl https://bootstrap.pypa.io/get-pip.py -o $VENV_DIR/get-pip.py
-  $VENV_PYTHONBIN $VENV_DIR/get-pip.py
+  curl https://bootstrap.pypa.io/get-pip.py | ${VENV_PYTHONBIN}
```
Comment (mmphego) [inline script/release/setup-venv.sh]: Refactored code by piping curl into Python.

**Pull request closed without merging**